[2023-07-30 22:05:24,693] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-23T00:00:00+00:00 [queued]>
[2023-07-30 22:05:24,818] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-23T00:00:00+00:00 [queued]>
[2023-07-30 22:05:24,818] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:24,818] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-30 22:05:24,819] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:24,961] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): dag_envio> on 2023-03-23 00:00:00+00:00
[2023-07-30 22:05:24,998] {standard_task_runner.py:52} INFO - Started process 1278 to run task
[2023-07-30 22:05:25,033] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_smtp_email_automatico', 'dag_envio', 'scheduled__2023-03-23T00:00:00+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/dag_smtp_email.py', '--cfg-path', '/tmp/tmpvp_k5w4i', '--error-file', '/tmp/tmpoamd56s6']
[2023-07-30 22:05:25,033] {standard_task_runner.py:80} INFO - Job 198: Subtask dag_envio
[2023-07-30 22:05:25,766] {task_command.py:371} INFO - Running <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-23T00:00:00+00:00 [running]> on host b0e2d990d7ae
[2023-07-30 22:05:26,655] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=dag_smtp_email_automatico
AIRFLOW_CTX_TASK_ID=dag_envio
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-23T00:00:00+00:00
[2023-07-30 22:05:28,154] {logging_mixin.py:115} INFO - (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials n188-20020a4a40c5000000b005634ef4f1c0sm1094020ooa.31 - gsmtp')
[2023-07-30 22:05:28,155] {logging_mixin.py:115} INFO - Failure
[2023-07-30 22:05:28,155] {python.py:173} INFO - Done. Returned value was: None
[2023-07-30 22:05:28,209] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_smtp_email_automatico, task_id=dag_envio, execution_date=20230323T000000, start_date=20230730T220524, end_date=20230730T220528
[2023-07-30 22:05:28,276] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-30 22:05:28,475] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
